openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KurentoHandler.java

```diff
@@ -112,6 +112,8 @@
 	private int watchThreadCount = 10;
 	@Value("${kurento.kuid}")
 	private String kuid;
+	@Value("${kurento.certificateType}")
+	private String certificateType;
```

Comment:
This can be `CertificateKeyType`

put `@Value("${kurento.certificateType}")` on setter
and perform conversion :)

the easy way `CertificateKeyType.valueOf(injectedString)`